client/src/store/modules/ehrDataStore.js

```diff
@@ -46,10 +46,18 @@ const getters = {
     const studentData = rootGetters['activityDataStore/assignmentData'] || {}
     let results = {}
     pageKeys.forEach( key => { 
+      const combinedObject = Object.assign({}, mergedData[key], seedData[key])
+      const isTable = Object.keys(combinedObject).includes('table')
```

Comment:
Is this check for 'table' fragile?  Might the work 'table' appear in the data?  Can you find a more robust way to detect the page has *one or more* tables.

Reply:
@bryan-gilbert, I have implemented it in a new way. I have divided into three main groups (1. It has one or more tables; 2. Has one or more tables and other content; 3. Has no tables). I have implemented it in a way that, it identifies which use cases it would fall into and implement a different logic for each. I think it improves the overall logic when it comes to the issues commented... I look forward to your thoughts on it. Thank you!